packages/components/src/VirtualizedList/utils/resizable.test.js

```diff
@@ -0,0 +1,325 @@
+			},
+		];
+		const currentIndex = 1;
+		// when
+		const ret = resizeColumns(deltaX, columnsWidths, currentIndex);
+		// then
+		expect(ret).toEqual([
+			{ dataKey: 'columnOne', resizable: true, resized: false, width: 70 },
+			{ dataKey: 'columnTwo', resizable: true, resized: true, width: 110 },
+			{ dataKey: 'columnThree', resizable: true, resized: true, width: 120 },
+		]);
+	});
+	it('should enlarge columnOne and shrink columnThree, skipping columnTwo which is at minimum width', () => {
+		// given
+		const deltaX = 10;
+		const columnsWidths = [
+			{
+				dataKey: 'columnOne',
+				width: 70,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnTwo',
+				width: 40,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnThree',
+				width: 130,
+				resizable: true,
+			},
+		];
+		const currentIndex = 0;
+		// when
+		const ret = resizeColumns(deltaX, columnsWidths, currentIndex);
+		// then
+		expect(ret).toEqual([
+			{ dataKey: 'columnOne', resizable: true, resized: true, width: 80 },
+			{ dataKey: 'columnTwo', resizable: true, resized: false, width: 40 },
+			{ dataKey: 'columnThree', resizable: true, resized: true, width: 120 },
+		]);
+	});
+	it('should enlarge columnTwo and shrink columnFour, skipping columnThree which is not resizable', () => {
+		// given
+		const deltaX = 10;
+		const columnsWidths = [
+			{
+				dataKey: 'columnOne',
+				width: 70,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnTwo',
+				width: 150,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnThree',
+				width: 120,
+				resizable: false,
+			},
+			{
+				dataKey: 'columnFour',
+				width: 130,
+				resizable: true,
+			},
+		];
+		const currentIndex = 1;
+		// when
+		const ret = resizeColumns(deltaX, columnsWidths, currentIndex);
+		// then
+		expect(ret).toEqual([
+			{ dataKey: 'columnOne', resizable: true, resized: false, width: 70 },
+			{ dataKey: 'columnTwo', resizable: true, resized: true, width: 160 },
+			{ dataKey: 'columnThree', resizable: false, resized: false, width: 120 },
+			{ dataKey: 'columnFour', resizable: true, resized: true, width: 120 },
+		]);
+	});
+	it('should shrink columnTwo and enlarge columnThree', () => {
+		// given
+		const deltaX = -10;
+		const columnsWidths = [
+			{
+				dataKey: 'columnOne',
+				width: 70,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnTwo',
+				width: 100,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnThree',
+				width: 130,
+				resizable: true,
+			},
+		];
+		const currentIndex = 1;
+		// when
+		const ret = resizeColumns(deltaX, columnsWidths, currentIndex);
+		// then
+		expect(ret).toEqual([
+			{ dataKey: 'columnOne', resizable: true, resized: false, width: 70 },
+			{ dataKey: 'columnTwo', resizable: true, resized: true, width: 90 },
+			{ dataKey: 'columnThree', resizable: true, resized: true, width: 140 },
+		]);
+	});
+	it('should shrink columnOne and enlarge columnThree, skipping columnTwo which is at minimum width', () => {
```

Comment:
This is not a wanted behavior. When we resize columnOne, columnTwo that is resizable should be enlarged.
Same for next test